CHANGELOG.md

```diff
@@ -13,6 +13,17 @@ before starting to add changes. Use example [placed in the end of the page](#exa
 
 - [S2FRMS-40] Adding child select autopopulate fields
 
+## [3.15.4] 2024-07-08
+
+- [#117](https://github.com/OS2Forms/os2forms/pull/117)
+  Encrypts all elements if encryption enabled.
+- [#114](https://github.com/OS2Forms/os2forms/pull/114)
+  Encrypted computed elements.
+
+## [3.15.3] 2024-06-25
+
+- [OS-74] Replacing DAWA matrikula select with Datafordeler select
+
```

Comment:
These changes do not seem related to this pull request.

Reply:
these changes are not visible when you look at the pull request changes: into develop from f/S2FRMS-40_child_select:
https://github.com/OS2Forms/os2forms/pull/111/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

It is in fact a resolution of the conflict we had in CHANGELOG.md file between two branches. If this conflict would have been resolved otherwise (meaning deleting these 10 lines), then upon merging these 10 lines would also be deleted from develop branch.

Reply:
Oops, my bad. Thanks for the explanation.